src/overlays/actors/ovl_En_Dh/z_en_dh.c

```diff
@@ -4,15 +4,7 @@
 
 #define THIS ((EnDh*)thisx)
 
-typedef enum {
-    DH_WAIT,
-    DH_RETREAT,
-    DH_BURROW,
-    DH_WALK,
-    DH_ATTACK,
-    DH_DEATH,
-    DH_DAMAGE
-} EnDhAction;
+typedef enum { DH_WAIT, DH_RETREAT, DH_BURROW, DH_WALK, DH_ATTACK, DH_DEATH, DH_DAMAGE } EnDhAction;
```

Comment:
This enum needs value comments for proper vertical formatting. (We should really just have value comments in all enums imo)